pkg/operator/controllers/routefix/routefix_controller.go

```diff
@@ -45,7 +45,7 @@ type Reconciler struct {
 
 var (
 	verFixed47, _ = version.ParseVersion("4.7.18")  // fixed in 4.7.18
-	verFixed46, _ = version.ParseVersion("4.6.37 ") // fixed in 4.6.37
```

Comment:
`version.ParseVersion("4.6.37 ")` won't be equal `version.ParseVersion("4.6.37")` as space will be parsed a `Suffix`.